packages/repo-mock/src/index.ts

```diff
@@ -39,16 +41,18 @@ export function mockRepo(
   const rootMeta = createRootMeta(keyPair);
 
   // Calculate paths for all of the metadata files
-  const rootPath = `${metadataPrefix}/1.root.json`;
+  const rootPath = `${metadataPrefix}/2.root.json`;
```

Comment:
why 2.root.json?

Reply:
We never request `1.root.json` since it will already be in the cache (the initial, trusted root). The Updater will only look for `2.root.json` to see if there is an updated root available.

Reply:
aww yeah... user copy the local root.json as 1.root.json. so the updater only look for anything greater than 1.root.json.

https://github.com/theupdateframework/tuf-js/blob/c4291861203d5d297ae9fd96b242343a4b816a23/packages/client/src/updater.ts#L159-L163